POS/UserInterface/01a_Cashpoint.md

```diff
@@ -9,82 +9,139 @@
+    The offer is added to the bill list.      
+
+- [CANCEL]   
+   Click this button if the customer is too young to buy this offer.   
+   The offer is not added to the bill list. 
+
+
+
+<!---NEU-->
+## Choose options
+
+*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT > Select offer*
+
+![Choose options](../../Assets/Screenshots/POS/Sales/Cashpoint/ChooseOption2.png "[Choose option]")
+
+The *Choose options* window is displayed if a selected offer has additional product options that represent a service. For detailed information, see [Add a product option](../Operation/02_SelectOffer.md#add-a-product-option). 
+
+- Additional field   
+    Enter additional information to a single option, if applicable. This can be a text or a name that should be printed on a shirt or another item.
+
+- [CANCEL]   
+    Click this button if the customer no longer wants to purchase the offer associated with the product option.
+
+- [SAVE]   
+    Click this button if the customer wants to purchase the offer, or both the offer and the product option.   
+    If you previously selected a product option, both the associated offer and the product option are added to the bill list.
+    If you have not previously selected a product option, only the associated offer is added to the bill list.
+
+
+
+<!---NEU-->
+## Suggest an offer to the customer
+
+*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT > Select offer > Click [PAY]*
+
+ ![Suggest offers](../../Assets/Screenshots/POS/Sales/Cashpoint/ProductReminder.png "[Suggest offer]")
+
+ - [PROCEED]  
+   Click this button after you have suggested the offer to the customer. For detailed information, see [Suggest an offer to the customer](../Operation/04_CompletePurchase.md#suggest-an-offer-to-the-customer).
 
-    > [Info] For detailed information, see [Create a QuickSelect Category](../Integration/07_ManageOffers.md#create-a-quickselect-category).
 
 
 ## Bill list
 
-*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT*
+*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT > Select offer*
 
 ![Bill List](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/BillList.png "[Bill List]")
+<!---NEU-->
+- ![Customer details](../../Assets/Icons/Scan.png "[Customer details]")(Customer details)      
+    Click this button to scan a customer code with a barcode scanner or to enter the customer number manually, see [Assign a purchase via customer number](../Operation/03_EditBillList.md#assign-a-purchase-via-customer-number).
 
-- ![Add Customer](../../Assets/Icons/Customer02.png "[Add Customer]") (Add Customer)   
-    Click this button to assign the bill to a certain customer. The *Add Customer* view is displayed.
+- ![Add customer](../../Assets/Icons/Customer02.png "[Add customer]") (Add Customer)   
+    Click this button to assign the bill to a certain customer. The *Add customer* view is displayed, see [Add customer](#add-customer).
 
 - *Customer Name and Address*   
```

Comment:
kein kursiv, es gibt kein Feld, das so heißt (es wird einfach Customer name and address angezeigt)

Reply:
done